### DISCUSSION TITLE
Step 2

### DISCUSSION THREAD

**Discussion by DDMAIN** [skills/.github#328] [Test with Actions] [answered]
When the tutorial said this

### :keyboard: Activity: Fix the test

1. Update the code in the `ci` branch to get the test to pass. You need to look something like this:
   ```markdown
   _underscore_
   ```

What file does the tutorial refer to?

Thanks

**Comment by sinsukehlab** [comment, marked as answer]
This instruction is really confusing.
Look at the Actions logs for the error message. The message following $\color{#d4a72c}{\textsf{warning}}$ helps a lot.

**Reply by DDMAIN** [reply]
Thank you so much. I got it

**Reply by sinsukehlab** [reply]
You're welcome.

**Comment by hyphena** [comment]
Commenting for posterity's sake as I was convinced GitHub actions had broken yet again like on previous tutorials. This is the intended output:

```js
Run markdown lint

----------------------------------------------------------------
> Run npm install remark-cli remark-preset-lint-consistent

added 202 packages in 8s

103 packages are looking for funding
  run `npm fund` for details
.github/steps/0-welcome.md: no issues found
.github/steps/1-add-a-test-workflow.md: no issues found
.github/steps/2-fix-the-test.md: no issues found
.github/steps/3-upload-test-reports.md: no issues found
.github/steps/4-add-branch-protections.md: no issues found
.github/steps/[5](https://github.com/hyphena/skills-test-with-actions/actions/runs/8696310314/job/23849272201#step:3:6)-merge-your-pull-request.md: no issues found
.github/steps/X-finish.md: no issues found
README.md: no issues found
resume.md
9:18-9:23    warning Unexpected emphasis marker `*`, expected `_`                      emphasis-marker remark-lint
  [cause]:
    3:1-3:[7](https://github.com/hyphena/skills-test-with-actions/actions/runs/8696310314/job/23849272201#step:3:8)9 info    Emphasis marker style `'_'` first defined for `'consistent'` here emphasis-marker remark-lint

⚠ 1 warning
Error: Process completed with exit code 1.
```
I'm not familiar with this specific kind of error output just yet so I was caught off guard lol. It does point out where and why your test has failed, in this case your repo contains a  `resume.md` file that had an improperly formatted line that uses *'s instead of _'s. Update it real quick and you're good to go.

**Reply by 12Gitcat** [reply]
1

**Comment by devopsvaleriano** [comment]
Good inputs, the syntax error was in resume.md file, at * Git *
 Just needed to change it for _ Git _

Thanks
